gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/query/HiveAvroORCQueryGenerator.java

```diff
@@ -355,12 +354,24 @@ private static Properties getTableProperties(Properties tableProperties) {
     // .. hence specifying 'use dbName' as a precursor to rename
     // Refer: HIVE-2496
     ddls.add(String.format("USE %s%n", dbName));
-    ddls.add(String.format("ALTER TABLE `%s` ADD IF NOT EXISTS %s LOCATION '%s' ", tableName, partitionSpecs,
-        partitionLocation));
+    if (format.isPresent()) {
```

Comment:
Format should also be validated to ensure its supported to avoid bad format name being set. We should not rely on Hive to complain because different versions of Hive behave differently in face of syntactic / query grammar issues.